akka-samples/akka-docs-java-lambda/src/test/java/util/SocketUtils.java

```diff
@@ -0,0 +1,27 @@
+package util;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.nio.channels.ServerSocketChannel;
+
+public class SocketUtils {
+
+  public static InetSocketAddress temporaryServerAddress(String hostname) {
```

Comment:
@patriknw did you create an issue for it? I couldn't find it (moving `temporaryServerAddress` to `actor-testkit`).